frontend/src/views/StartView.vue

```diff
@@ -40,5 +49,8 @@ if (
   (await validateUser())
 ) {
   setupProjectId()
+  if (await validateUser()) {
+    router.push('/collection')
+  }
```

Comment:
When client has a valid user cookie, app will redirect to the Collection View.
I'm not sure if it the right logic.